jscomp/common/js_config.ml

```diff
@@ -51,15 +55,41 @@ let cmi_only = ref false
 let cmj_only = ref false
 let force_cmi = ref false
 let force_cmj = ref false
-let jsx_version = ref (-1)
-let jsx_module = ref "react"
-let jsx_mode = ref "classic"
+let jsx_version = ref NotSelected
+let jsx_module = ref React
+let jsx_mode = ref Classic
 let js_stdout = ref true
 let all_module_aliases = ref false
 let no_stdlib = ref false
 let no_export = ref false
 let as_ppx = ref false
 
+let int_of_jsx_version = function
```

Comment:
We don't need this right?

Reply:
If we parse directly into the variant.

Reply:
This function is one of encode/decode functions for the variant. Isn't it better to have it?